### ISSUE TITLE
Release new version

### ISSUE THREAD

**Issue by stephanlachnit** [kokkos/mdspan#299]
Could you tag a new version on GitHub? The last 0.6 release lacks the `MDSPAN_IMPL_*_NAMESPACE` macros.

**Comment by crtrott** [comment]
Yeah we are trying to stabilize a version right now, its been taking a bit longer than we hoped for.
